apm-sniffer/apm-sdk-plugin/aerospike-plugin/src/main/java/org/apache/skywalking/apm/plugin/aerospike/AerospikeClientMethodInterceptor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.skywalking.apm.plugin.aerospike;
+
+import org.apache.skywalking.apm.agent.core.context.ContextManager;
+import org.apache.skywalking.apm.agent.core.context.tag.Tags;
+import org.apache.skywalking.apm.agent.core.context.trace.AbstractSpan;
+import org.apache.skywalking.apm.agent.core.context.trace.SpanLayer;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceMethodsAroundInterceptor;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
+import org.apache.skywalking.apm.network.trace.component.ComponentsDefine;
+
+import java.lang.reflect.Method;
+
+public class AerospikeClientMethodInterceptor implements InstanceMethodsAroundInterceptor {
+    @Override
+    public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
+                             MethodInterceptResult result) throws Throwable {
+        String peer = String.valueOf(objInst.getSkyWalkingDynamicField());
+        AbstractSpan span = ContextManager.createExitSpan("Aerospike/" + method.getName(), peer);
+        span.setComponent(ComponentsDefine.AEROSPIKE);
+        Tags.DB_TYPE.set(span, "Aerospike");
+        SpanLayer.asCache(span);
```

Comment:
I think these two are not aligned. You should either use cache or database layer with the tags
1. Virtual Database, https://skywalking.apache.org/docs/main/v9.5.0/en/setup/service-agent/virtual-database/
2. Virtual Cache, https://skywalking.apache.org/docs/main/v9.5.0/en/setup/service-agent/virtual-cache/

Metrics are generated according.